.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [14, 16, 18]
+        node: [14, 16, 18, 20]
```

Comment:
can we set this up with an action that can keep it up to date automatically? i think pkgjs has one, but if not, i have ljharb/actions/node/matrix

Reply:
I am aiming pretty low since there is a suggestion to archive repo: https://github.com/pkgjs/parseargs/issues/149#issuecomment-1508487614

(Thanks for the review! I wasn't certain I would get one.)